core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/html/dot-seo-meta-tags.service.ts

```diff
@@ -153,6 +113,46 @@ export class DotSeoMetaTagsService {
             .sort((a, b) => a.sort - b.sort);
     }
 
+    /**
+     * This returns the map of the open graph elements and their properties.
+     */
+    private openGraphMap(): OpenGraphOptions {
```

Comment:
Instead of just`OpenGraphOptions` you can do something like:

```typescript
enum myEnum {
    myKey = 'myKey'
}

interface OpenGraphOptions {
    getItems: () => any;
    sort: number;
    info: string;
}

function openGraphMap(): Record<myEnum, OpenGraphOptions> {
    return {
        [myEnum.myKey]: {
            getItems: () => {
                return 0;
            },
            sort: 1,
            info: 'myInfo'
        }
    };
}

```

This way if the array of keys is an array of the enum, you won't need to null check the methods/properties